src/main/java/org/prebid/server/bidder/rubicon/RubiconBidder.java

```diff
@@ -169,6 +169,7 @@ private BidRequest createSingleRequest(Imp imp, BidRequest bidRequest) {
                 .site(makeSite(bidRequest.getSite(), rubiconImpExt))
                 .app(makeApp(bidRequest.getApp(), rubiconImpExt))
                 .imp(Collections.singletonList(makeImp(imp, rubiconImpExt)))
+                .cur(makeCurrencies(bidRequest.getCur()))
```

Comment:
As far as I understand `$.cur` should be unconditionally removed from the request.